test/validate/eia_test.py

```diff
@@ -45,16 +45,16 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_473_364, 1_473_364, 123_369),
+        ("bf_eia923", 1_521_304, 1_521_304, 127_412),
         ("bga_eia860", 141_638, 141_638, 141_638),
         ("boil_eia860", 80_059, 80_059, 80_059),
-        ("frc_eia923", 584_724, 241_751, 23_737),
-        ("gen_eia923", None, 4_765_282, 398_747),
+        ("frc_eia923", 631_975, 258_115, 25_370),
+        ("gen_eia923", None, 5_178_892, 433_286),
         ("gens_eia860", 556_578, 556_578, 556_578),
-        ("gf_eia923", 2_598_272, 2_598_272, 222_356),
+        ("gf_eia923", 2_859_269, 2_859_269, 244_796),
         ("own_eia860", 89_687, 89_687, 89_687),
         ("plants_eia860", 200_177, 200_177, 200_177),
-        ("pu_eia860", 180_342, 180_342, 180_342),
+        ("pu_eia860", 199_332, 199_332, 199_332),
```

Comment:
Can you look up the expected number of rows that were being used on `dev` prior to the previous PR getting merged, so we can compare directly between the new expectations, and the expectations we had prior to the changes that resulted in the loss of all these rows? That will let us evaluate the net impacts of the ownership harvesting changes and see if there's anything suspicious that should still be investigated / compared with the last good `dev` output.  Here and in the `mcoe_test` outputs below.

Reply:
Yep, should have said this earlier but these row counts as well as MCOE are the same as in `dev` except for the `utils_eia860` table, which has all the extra owner utility rows.

Reply:
Oh great, good to know!